.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: 'release'
```

Comment:
```suggestion
name: release
```

Reply:
:rofl: 
If this is the only comment, I am very happy.